source/windows/clock.c

```diff
@@ -59,8 +57,8 @@ int aws_high_res_clock_get_ticks(uint64_t *timestamp) {
     LARGE_INTEGER ticks, frequency;
     /* QPC runs on sub-microsecond precision, convert to nanoseconds */
     if (QueryPerformanceFrequency(&frequency) && QueryPerformanceCounter(&ticks)) {
```

Comment:
QueryPerformanceFrequency() returns the same result for the life of the program, see [docs](https://msdn.microsoft.com/en-us/library/windows/desktop/ms644904(v=vs.85).aspx). You could save the value in a static int.

Reply:
It's static under the hood anyways. and it's not really related to this change.